akamai.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	// VERSION Application Version
-	VERSION = "1.0.2"
+	VERSION = "1.0.3"
```

Comment:
Don't bump the version, we only do this right before release; this is creating a merge conflict.